blamo/lib/Test/tests.dart

```diff
@@ -206,7 +206,53 @@ class _TestsPageState extends State<TestsPage> {
   }
 
   void _onTileLongClicked(int i) async {
+    String result;
+    result = await showDialog(
+        context: context,
+        builder: (context) => AlertDialog(
+          title: Text("Are you sure you want to delete ${currentState.testList[i]}?"),
```

Comment:
What do you think about showing the test bounds as displayed on the tests page instead of the test name? My thought is that a test name doesn't necessarily correlate to what the user is trying to delete, so validation could be a lil sketchy in that regard